docs/tutorials/Canopy/canopy_tutorial.jl

```diff
@@ -0,0 +1,365 @@
+# # Introduction to the Canopy Model
+
+# This tutorial shows how to instantiate and run a simulation of the
+# canopy biophysics model in ClimaLSM. A 
+# ['CanopyModel'](@ref ClimaLSM.Canopy.CanopyModel) including all component 
+# models is initialized, then an example simulation is run. The initial conditions,  
+# atmospheric and radiative flux conditions, and canopy properties are set up 
+# to match those observed at the US-MOz flux tower, a flux tower located within 
+# an oak-hickory forest in Ozark, Missouri, USA. See Wang et al. 2021 
+# (https://doi.org/10.5194/gmd-14-6741-2021) for details on the site and
+# canopy parameters.
+
+# The canopy biophysics model in ClimaLSM couples a photosynthesis model with a 
+# canopy radiative transfer scheme and plant hydraulics model, placing them 
+# under either prescribed or simulated atmospheric and radiative flux 
+# conditions. 
+
+# Currently, the only radiative tansfer model supported in ClimaLSM is 
+# Beer-Lambert law, in which the intensity of light absorbed is a negative 
+# exponential function of depth in the canopy and an exinction coefficient
+# determined by optical depth. For details on the Beer-Lambert model see 
+# (TODO: Dynamic Doc link).
```

Comment:
could we remove the TODO comments? We can add the sentence back in when the dynamic docs are live.